### DISCUSSION TITLE
Inability to Access "Configure CAS" Sub-menu in LabPlot Software and access R notebook

### DISCUSSION THREAD

**Discussion by ghost** [dat-linux/community#56] [Issues]
Issue Description:
I am experiencing difficulty accessing the "Configure CAS" sub-menu under the settings menu in LabPlot software. Therefore, i am unable to open or create tiers software computational notebook (specifically R notebook option, as well as python notebook)

Do you have any idea ???

PS: labplot 2.10.0 is currently installed (but surprisingly the system info IS showing labplot 2.9.0 as the version installed). I uninstalled, re-installed back and rebooted the system, but no change.

Thanks

**Comment by dat-linux** [comment]
Is there supposed to be a sub-menu called "Configure CAS"? This is not standard.

Re the second issue, a patch will be released to fix this irregularity.

**Comment by dat-linux** [comment]
Actually the version irregularity is due to the package version on the system not reflecting the version indicated in the software itself. So I wouldn't worry about it causing a problem as such.

But we'll be switching this to an AppImage installer instead, which should just show up as an upgrade in the control panel admin "Apps Updater". Give it a couple days.

**Comment by ghost** [comment]
**> Is there supposed to be a sub-menu called "Configure CAS"? This is not standard.**

>  although it might not be standard, you may check the following path: File -> add new -> notebook -> !!! 

Not able to open or create a notebook, which is odd

It would be really helpful if you could provide some help, thank you.


**> Re the second issue, a patch will be released to fix this irregularity.**

> sure i will wait for a couple of days and update the system.

**Comment by dat-linux** [comment]
Unfortunately there is a font issue in the Appimage. Hopefully it's a simple matter of installing a font. Waiting on feedback from the devs.
![screen](https://github.com/dat-linux/community/assets/99078210/ce5e8be1-1081-426e-91c3-154436b604f5)

**Reply by dat-linux** [reply]
You can add new workbook, etc, by right clicking on a project
